docs/samples/customdomain/README.md

```diff
@@ -0,0 +1,134 @@
+# Setting up custom domain name
+
+## Setup
+
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/blob/master/docs/DEVELOPER_GUIDE.md#deploy-kfserving).
+2. Your cluster's Istio Ingress gateway must be network accessible.
+3. You have a custom domain configured to route incoming traffic to the cluster's Ingress gateway.
+
+## Create the Ingress resource
+
+Edit the `knative-ingress.yaml` file to add your custom wildcard domain to the `spec.rules.host` section, replacing `<*.custom_domain>` with your custom wildcard domain. This is so that all incoming network traffic from your custom domain and any subdomain is routed to the `kfserving-ingressgateway`.
```

Comment:
In the set up you said the Istio Ingress gateway must be network accessible, but here you are using the `kfserving-ingressgateway`. Maybe for consistency, change `kfserving-ingressgateway` to `istio-ingressgateway` and add a comment to clarify all the kfserving traffic is routing to `kfserving-ingressgateway` if it's deployed with the latest Kubeflow stack.